mysqlnd_azure.c

```diff
@@ -654,13 +689,15 @@ MYSQLND_METHOD(mysqlnd_azure, connect)(MYSQLND * conn_handle,
 
                     ret = (*pconn)->m->connect(pconn, redirect_host, redirect_user, password, database, redirect_info->redirect_port, socket_or_pipe, mysql_flags);
                     if (ret == FAIL) {
+                        AZURE_LOG(ALOG_LEVEL_INFO, "Redirect cache miss");
```

Comment:
This should bot be "Redirect cache miss" since cache already found, but "Use redirect cache failed to connect"